app/models/fleet.rb

```diff
@@ -22,38 +26,37 @@ def cargo
 
   def available_capacity
     loaded = 0
-    self.cargo.each { |cargo| loaded += cargo.unit.weight * cargo.quantity }
-    (self.capacity * self.quantity) - loaded
+    cargo.each { |cargo| loaded += cargo.unit.weight * cargo.quantity }
+    (capacity * quantity) - loaded
   end
 
   def weight
-    self.quantity * self.unit.weight
+    quantity * unit.weight
   end
 
-  def load_in(carrier,quantity)
-    if self.unit.weight * quantity > carrier.available_capacity
-      quantity = carrier.available_capacity  / self.unit.weight
+  def load_in(carrier, quantity)
+    if unit.weight * quantity > carrier.available_capacity
+      quantity = carrier.available_capacity / unit.weight
       return nil if quantity < 1
     end
     if self.quantity == quantity
-      self.update_attributes(carrier: carrier, destination: carrier.destination, arrives_in: carrier.arrives_in)
+      update_attributes(carrier: carrier, destination: carrier.destination, arrives_in: carrier.arrives_in)
     else
-      left_behind = self.dup
+      left_behind = dup
       left_behind.quantity = self.quantity - quantity
       left_behind.save
-      self.update_attributes(quantity: quantity, carrier: carrier, destination: carrier.destination, arrives_in: carrier.arrives_in)
+      update_attributes(quantity: quantity, carrier: carrier, destination: carrier.destination, arrives_in: carrier.arrives_in)
     end
   end
 
-  def unload_from(carrier,quantity)
+  def unload_from(_carrier, quantity)
```

Comment:
It's weird you don't need the carrier here. Check if you're not forgetting anything.

Reply:
it do needs carrier! It seems to be a rubocop little misunderstanding. fixed.